packages/strapi-admin/admin/src/strapi.js

```diff
@@ -29,34 +29,35 @@ const registerPlugin = (plugin) => {
   merge(translationMessages, plugin.translationMessages);
 
   plugin.leftMenuSections = plugin.leftMenuSections || [];
-  const shouldAllowRegister = isPluginAllowedToRegister(plugin);
+  store.dispatch(pluginLoaded(plugin));
+  // const shouldAllowRegister = isPluginAllowedToRegister(plugin);
```

Comment:
These line should not be removed at the moment since we need them in case we want to switch to the old AdminPage container...